composer.json

```
@@ -12,7 +12,7 @@
     "symfony/http-kernel": "^4.1",
     "php-http/guzzle6-adapter": "^1.1",
     "symfony/serializer": "^4.4",
-    "symfony/property-access": "^4.1"
+    "symfony/property-access": "^4.1 || ^5.2"
```

I believe there is no reason to disallow versions 5.0 and 5.1. Also could you please add `|| ^5.0` to all `symfony/*` packages? Let's allow newer versions once and for good :)

```suggestion
    "symfony/property-access": "^4.1 || ^5.0"
```